.circleci/config.yml

```diff
@@ -106,7 +106,7 @@ defaults:
 
   - test_ubuntu1904: &test_ubuntu1904
       docker:
-        - image: ethereum/solidity-buildpack-deps:ubuntu1904
+        - image: ethereum/solidity-buildpack-deps:ubuntu1904_evmc7
```

Comment:
Where is the dockerfile for this image?

Reply:
The `ubuntu_clang` docker image used by the fuzzer should also be updated to make use of evmc7. To do this, please change this line of code in this PR as appropriate

https://github.com/ethereum/solidity/blob/5423dd6a485cf46077caf76ad338164a9758bce5/.circleci/docker/Dockerfile.clang.ubuntu1904#L119

Reply:
Also, if evmc7 requires newer versions of intx and ethash, please update them here

https://github.com/ethereum/solidity/blob/5423dd6a485cf46077caf76ad338164a9758bce5/.circleci/docker/Dockerfile.clang.ubuntu1904#L107

https://github.com/ethereum/solidity/blob/5423dd6a485cf46077caf76ad338164a9758bce5/.circleci/docker/Dockerfile.clang.ubuntu1904#L95

Reply:
I expect the fuzzer build to pass if the clang docker image is updated with newer evmone and its dependencies